.changeset/four-cameras-dream.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fixed: string formatter handling of log symbols
```

Comment:
[suggest] How about this for consistency and clarity? 👇🏼 

```suggestion
Fixed: `string` formatter handling of log symbols for non-Unicode environments 
```

Reply:
```suggestion
Fixed: `string` formatter handling of log symbols for non-Unicode terminals 
```

Reply:
Sounds good 👍🏼